src/Nancy.Siren.Demo/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
```

Comment:
This will need the dependencies for Kestrel, the new dotnet core web server. See here as an example to get started. https://github.com/NancyFx/Nancy/blob/master/samples/Nancy.Demo.Hosting.Kestrel/project.json